.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Hi @Michael-Obele 
any specific reason for this change ?

Reply:
This was supposed to be an internal setting for my VSCode; I will remove it.